docker.d/init.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+set -e
+
+# == START: This system variables are set by cloudops ==
+test $ENV  # ENV should be set to "dev", "stage" or "prod"
+test $COT_PRODUCT  # either "firefox" or "thunderbird"
+test $TASKCLUSTER_CLIENT_ID
+test $TASKCLUSTER_ACCESS_TOKEN
+test $ED25519_PRIVKEY
```

Comment:
`ED25519_PRIVKEY` can be unset in staging. It's not required when we don't sign the CoT files